platform/viewer/src/App.js

```diff
@@ -114,52 +117,61 @@ class App extends Component {
       );
     }
 
-    _initExtensions(this.props.extensions);
-    _initServers(this.props.servers);
+    this.appConfig = props || {};
```

Comment:
Would props ever be `undefined`? If so, the app wouldn't be able to run without at least some default values (server). Should we check for those and throw/show warnings that are helpful to the user so they know where to look for configuration options?

Reply:
alright.. I will remove it! When time comes and we split passed props into (properties appConfig, servers, extensions...) that part will make more sense..